mavlink/modules/flight_controller.py

```diff
@@ -253,3 +253,25 @@ def get_flight_mode(self) -> "tuple[bool, drone_odometry.FlightMode | None]":
         if flight_mode == "AUTO":
             return True, drone_odometry.FlightMode.MOVING
         return True, drone_odometry.FlightMode.MANUAL
+
+    def download_commands(self) -> "list[dronekit.Command]":
+        """
+        Downloads the current list of commands from the drone.
+
+        Returns
+        -------
+        list[dronekit.Command]
+            The list of commands currently held by the drone.
+        """
+        try:
+            command_sequence = self.drone.commands
+            command_sequence.download()
+            command_sequence.wait_ready()
+            commands = list(command_sequence)
+            return commands
```

Comment:
you should return whether the operation was successful with a bool

so the function should return tuple[bool, List[dronekit.Command]]